src/lib/index.ts

```diff
@@ -405,10 +417,18 @@ export class SourceFileConverter {
           } else {
             commands.push(new ast.RawText(`<${node.tagName}`));
           }
-          for (const {name, value} of node.attrs) {
-            const textLiterals = value.split(markerRegex);
-            if (
-              name.startsWith('.') ||
+          for (let {name, value} of node.attrs) {
+            if (name.startsWith('.')) {
+              // Need to get name from source to ensure proper casing.
+              const propertyName = lastAttributeNameRegex.exec(strings[partTypes.length])![2];
+              const reflectedName = reflectedAttributeName(propertyName.slice(1), node.tagName);
+              if (reflectedName !== undefined) {
+                name = reflectedName;
+              } else {
+                partTypes.push('attribute');
```

Comment:
I think we need to create a part for property expressions too... I need to think why this isn't messing anything up.